tasks/IsTheRefrigeratorRunning.tex

```diff
@@ -0,0 +1,37 @@
+\section{Is the Refrigerator Running [Housekeeper]}
+The people who live in the arena are so sloppy! The robot finds minor faults in the state of the arena and rectifies them. This is an advanced version of Clean Up for Stage II.
```

Comment:
From here I thought the fridge was broken and I got a robot-technician to fix it.

# Task Title: Tidy up the apartment
# Description:
The robot patrols the apartment and fixes up the mess.